test/reducers/columnStateHelper-test.js

```diff
@@ -303,68 +306,68 @@ describe('columnStateHelper', function() {
 
     it('should scroll to column when specified via prop', function() {
       oldState.scrollX = 0;
+      availableWidth = 350;
       const result = columnStateHelper.initialize(oldState, {
         scrollToColumn: 2,
-        width: 350,
       }, {});
 
       assert.deepEqual(result, Object.assign({}, oldState, {
-        maxScrollX: 400,
```

Comment:
These were wrong before but because columnStateHelper was using both props & state for getting width at different places & the test was only stubbing one, it was testing a bad state.